src/simplewallet/simplewallet.cpp

```diff
@@ -63,6 +63,7 @@
 #include "ringct/rctSigs.h"
 #include "wallet/wallet_args.h"
 #include <stdexcept>
+#include "cryptonote_config.h"
```

Comment:
I don't think we need to import the cryptonote_config here since the DEFAULT_MIXIN can be defined locally in simplewallet.cpp (line 83). We just need to assure that mixin = 1 is enforced at node level.